widgets/BigQueryTable-batchsource.json

```diff
@@ -96,6 +96,25 @@
           "widget-attributes": {
             "placeholder": "Google Cloud Storage bucket for temporary data"
           }
+        },
+        {
+          "name": "tempTableExpirationTimeUnit",
+          "label": "Temporary Table Expiration Units",
+          "widget-type": "select",
+          "widget-attributes": {
+            "default": "Day",
+            "values": ["Day", "Hour"]
+          }
+        },
+        {
+          "name": "tempTableExpirationTimeNum",
+          "label": "Temporary Table Expiration Length",
+          "widget-type": "number",
+          "widget-attributes": {
+            "default": "1",
+            "min": "1",
+            "max": "100"
```

Comment:
Why is max 100?

Reply:
Its the default. I wanted to restrict numbers to this field and for whatever reason they need to go that high its available. We could set it to 24 factoring in that one could use 24 hours for a day and this would cover from 1 hour up to 24 days of configuration time.

Reply:
I see. We can keep it 30.